src/main/java/com/yana/internship/entity/Order.java

```diff
@@ -0,0 +1,83 @@
+package com.yana.internship.entity;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "app_order")
```

Comment:
Why we need prefix `app`?

Reply:
"order"(order by) is reserved word in SQL

Reply:
Eap, exactly. Then let's use `apartment_order`. `app` looks like `application`.

Reply:
Done